contactaccessrights/js/test/controllers/access-rights-ctrl_test.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'common/angularMocks',
+  'access-rights/controllers/access-rights-ctrl'
+], function () {
+  'use strict';
+
+  describe('AccessRightsCtrl', function () {
+    var ctrl, modalSpy;
+
+    beforeEach(module('access-rights.controllers'));
+    beforeEach(inject(function (_$controller_, _$rootScope_) {
+      CRM = {
+        vars: {
+          contactAccessRights: {
+            baseURL: ''
+          }
+        }
+      };
+      modalSpy = jasmine.createSpyObj('modalSpy', ['open']);
+      ctrl = _$controller_('AccessRightsCtrl', {
+        $scope: _$rootScope_.$new(),
+        $modal: modalSpy
+      });
+    }));
+
+    describe('openModal', function () {
+      it('opens the modal', function () {
+        ctrl.openModal();
```

Comment:
if possible keep all the "set up" code in the `beforeEach` function, so that the `it` block only has the `expect` statement(s)

```js
describe('openModal', function ( ) {
  beforeEach(function () {
    ctrl.openModal();
  });

  it('opens the modal', function ( ) {
    expect(modalSpy.open).toHaveBeenCalled();
  });
});
```